applications/GeoMechanicsApplication/tests/cpp_tests/test_interface_stress_state_policy.cpp

```diff
@@ -0,0 +1,114 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "custom_elements/interface_stress_state.h"
+#include "custom_geometries/line_interface_geometry.h"
+#include "geo_mechanics_fast_suite.h"
```

Comment:
After moving the file, this include needs to be updated:
```suggestion
#include "tests/cpp_tests/geo_mechanics_fast_suite.h"
```

Reply:
Done